src/main/scala/uk/gov/hmrc/selenium/webdriver/Browser.scala

```diff
@@ -31,8 +33,9 @@ trait Browser extends FileDownload {
       val downloadDirectory = s"./target/test-reports/$filename/axe-results"
       val sessionId         = Driver.instance.getSessionId
 
-      Try(download(filename, downloadDirectory, sessionId))
+      Try(download(filename, downloadDirectory, sessionId)).failed
+        .foreach(logger.error("Error while download a11y reports", _))
```

Comment:
Can we update the message that's output to something like `Accessibility assessment: Failed to download results` for consistency please?